libc/utils/docgen/errno.json

```diff
@@ -0,0 +1,16 @@
+{
+  "macros": {
+    "EDOM": {
+      "c-definition": "7.5"
+    },
+    "EILSEQ": {
+      "c-definition": "7.5"
+    },
+    "ERANGE": {
+      "c-definition": "7.5"
+    },
+    "errno": {
+      "c-definition": "7.5"
+    }
+  }
```

Comment:
in future we'll probably need to add the posix and linux errno macros, but this is fine for now.

Reply:
Posix, definitely.

For Linux...I want to avoid having to document all of the UAPI headers.  I wonder if other OS' have additional errno.h extensions? idk, we'll have to think more about it for sure.